### DISCUSSION TITLE
能不能搞个电脑版出来？就随便搞搞都比别的浏览器强我觉得，能用就行。

### DISCUSSION THREAD

**Discussion by Glunefish** [tuyafeng/Via#1474] [Q&A]
能不能搞个电脑版出来？就随便搞搞都比别的浏览器强我觉得，能用就行。同步书签这些

**Comment by szb000801** [comment]
电脑版是没有webview的，还要调用浏览器内核，代码量很大

**Comment by Glunefish** [comment]
直接用谷歌内核啊



---原始邮件---
发件人: ***@***.***&gt;
发送时间: 2024年9月7日(周六) 中午12:58
收件人: ***@***.***&gt;;
抄送: ***@***.******@***.***&gt;;
主题: Re: [tuyafeng/Via] 能不能搞个电脑版出来？就随便搞搞都比别的浏览器强我觉得，能用就行。 (Discussion #1474)




 
电脑版是没有webview的，还要调用浏览器内核，代码量很大

**Comment by lijunlinjie** [comment]
套用edge同款内核就好了啊，或者chrome改点东西直接套用也可以啊
球球了，搞一个电脑版吧
